__tests__/app.test.js

```diff
@@ -0,0 +1,25 @@
+const app = require('../app.js')
+const request = require('supertest')
+const testData = require('../db/data/test-data/index')
+const seed = require('../db/seeds/seed')
+const connection = require('../db/connection')
+
+beforeEach(() => {
+    return seed(testData)
+})
+
+afterAll(() => {
+    return connection.end()
+})
+
+describe('/api/topics', () => {
+    test('200: array of topic objects returned with slug and description properties', () => {
+        return request(app).get('/api/topics')
+            .expect(200).then((response) => {
+                for (const obj of response.body) {
```

Comment:
Nice work using a loop here! Could we just add another line before the for loop which checks the length of the array coming back please?